schema/places/place.yaml

```diff
@@ -41,6 +41,23 @@ properties:
             items:
               "$ref": "./defs.yaml#/$defs/typeDefinitions/category"
             uniqueItems: true
+      basic_level_category:
```

Comment:
This is an array property (at least indirectly), so there are multiple of them. If you are really intending multiple values may I suggest we call this `basic_categories`?